src/ape/utils.py

```diff
@@ -41,6 +40,18 @@
 except ImportError:
     from singledispatchmethod import singledispatchmethod  # type: ignore
 
+if TYPE_CHECKING:
+    from ape.managers.accounts import AccountManager
+    from ape.managers.chain import ChainManager
+    from ape.managers.compilers import CompilerManager
+    from ape.managers.config import ConfigManager
+    from ape.managers.converters import ConversionManager
+    from ape.managers.networks import NetworkManager
+    from ape.managers.project import ProjectManager, _DependencyManager
```

Comment:
I think `_DependencyManager` should have the underscore removed since we are using it all over like this